release/mk-release-series.sh

```diff
@@ -49,9 +50,9 @@ fi
     git add doc/release-notes/release-${NEWVER}.sgml.in
 
   if `git diff HEAD` ; then
-    git commit -m "Branch ${NEWVER}.0.0"
+    git commit -m "${NEWVER}.0.0"
     git push
-    # github PR created by maintenance automation
+    gh pr create --repo squid-cache/squid --base master --fill
   fi
```

Comment:
> PR description: @rousskov requires that each change to squid-cache/squid repository by automation be submitted as a separate PR.

Sigh. I do not require that, so I removed the above statement from the PR description and synced the rest of the PR title/description the best I could. Please adjust further as needed.

If you would like to propose a new policy, please do so (there is nothing wrong with that per se!), but please do not assign authorship of your proposals to me.

Reply:
Er, what? the record is clear - you argued against and vetoed the alterations this script is/was committing to `vN-maintenance` branch automatically. So yes you did literally require this PR change by any reasonable definition of the word "requires".

What you apparently still have not grokked is that there are **many** maintenance scripts doing **many** changes to the source code. Most of them are not as visible as this one. All of them are currently committing their changes to the appropriate `vN-maintenance` branch for later (daily-ish) creation of a PR to merge the aggregate into the relevant vN series branch.

The maintenance update this PR is touching is "just" the one bit which will hopefully let us release the first v7.0.0 snapshot without you veto'ing the subsequent maintainer work.

Reply:
> PR description: @rousskov requires that each change to squid-cache/squid repository by automation be submitted as a separate PR.

> Alex: I do not require that.

> Amos: you argued against and vetoed the alterations this script is/was committing to vN-maintenance branch automatically. 

Not really. I [suggested](https://github.com/squid-cache/squid/pull/1268#discussion_r1097741975) some related improvements (e.g., which changes should, ideally, go into which PR), but I did not argue against the changes this script is committing (any polishing touches aside), and I certainly did not veto those alterations (I have never even voted against PR 1268!).

More importantly, all those sins that I have been falsely accused of do not actually imply that I require that "each change to squid-cache/squid repository by automation be submitted as a separate PR"! Some automation-created changes should be merged inside a single PR. Some should not be. Many variables affect that decision. I was not defining any new Project policy when I discussed specific improvements in a specific PR, and I certainly was not defining the policy summarized in the original version of this PR title!

> So yes you did literally require this PR change

I did not. It is impossible to logically connect anything I have _actually_ said or _actually_ did to this conclusion (not to mention to find a "literal" requirement for this PR change!). It is only possible to connect a yet another misinterpretation of what actually happened to the above conclusion. What a waste of time and nerve cells!